sedldata/cli.py

```diff
@@ -27,20 +50,26 @@ def upgrade():
 
 
 @cli.command()
-def load():
+@click.argument('infile')
+@click.argument('outfile')
+@click.option('--name', default=None)
+def load(infile, outfile, name):
+    if name is None:
+        name = infile
     # Load something into the database
     now = datetime.datetime.now()
-    i = metatable.insert()
-    i.execute(load_datetime=now, data_source="Test")
-    click.echo("Loaded test: %s" % now)
+    unflattened = xl_to_json(infile, outfile)
+    i = datatable.insert()
```

Comment:
Is this the standard way of naming a variable with an insert method? In case it is not, would it be possible to give it a meaningful name?